exporter/tanzuobservabilityexporter/config.go

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tanzuobservabilityexporter
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/config"
+)
+
+type TracesConfig struct {
+	// Wavefront Proxy URL of the form protocol://host[:port]
+	Endpoint string `mapstructure:"endpoint"`
```

Comment:
What is the protocol? HTTP? If yes we have a `confighttp` helper to keep everything synchronize.

Reply:
Yes, it's HTTP and we'll look at using `confighttp` in an update to this PR.

Reply:
Fixed to use `confighttp` in d5ef662.